WebHostLib/misc.py

```diff
@@ -49,12 +49,6 @@ def weighted_options():
     return render_template("weighted-options.html")
 
 
-# TODO for back compat. remove around 0.4.5
-@app.route("/games/<string:game>/player-settings")
-def player_settings(game: str):
-    return redirect(url_for("player_options", game=game), 301)
-
-
```

Comment:
I was unsure whether or not to remove the backwards compatability in this PR, but it seems fitting to do so at this point